js/modules/k6/html/element.go

```diff
@@ -246,6 +246,18 @@ func (e Element) IsSameNode(v goja.Value) bool {
 	return false
 }
 
+// Selection returns a Selection object based on the current Element.
+//
+// This function is used to create a Selection object that represents the same HTML
+// content as the Element. It is useful for performing operations or manipulations
+// on the HTML content within the scope of this Element.
+//
+// Example:
+// sel := element.Selection()
+func (e Element) Selection() Selection {
+	return Selection{rt: e.sel.rt, sel: e.sel.sel, URL: e.sel.URL}
+}
```

Comment:
```suggestion
func (e Element) Selection() Selection {
	return *e.sel
}
```

I think this would be slightly better in terms of maintainability and potential compiler optimizations, and the only drawback I see is in case `e.sel` is `nil, but that would need special handling in the current code as well.

Reply:
thanks!